atlas/local-app-development/frontend-app-config.mdx

```diff
@@ -62,20 +79,46 @@ NEXT_PUBLIC_WORDPRESS_URL=https://my-site.wpengine.com
 
-As an example, if your project has a `NEXT_PUBLIC_WORDPRESS_URL` environment variable and your local WordPress site URL is `http://my-site.local`, then that environment variable should be updated to `NEXT_PUBLIC_WORDPRESS_URL=http://my-site.local`.
+Once your dependencies have finished installing, you can run the command to start your JavaScript app in development mode. If your site was created with the Faust.js starter, the command `npm run dev` will start your app in development mode and make it available at `localhost:3000` in a web browser. 
 
-When finished, save the environment variables file.
+If you are using another framework, you will need to reference that framework's docs or `package.json` file.
 
-## 3. Install Dependencies and Start App
+## 4. Push Changes to Atlas
 
-Run `npm install` to install the frontend app's NPM dependencies.
+With your site running in development mode, the app running in the browser is sourcing your local JavaScript files for the frontend, but still pulling data from your remote WordPress install on WP Engine's servers. In the next step of this guide, you will learn how to create a separate WordPress instance locally to isolate your development work, but now is a great time to test your connection to GitHub and the Atlas Platform.
 
-Make sure your local WordPress site is up and running, then run the command to start your JavaScript app in development mode (typically `npm run dev`).
+Make some basic changes to your site locally: add a component, change the way posts or pages are displayed or update the site styles with your own CSS.
+
+Once you are ready to push your changes, you can trigger a rebuild of your site by pushing to GitHub. A basic workflow assuming SSH configuration would look like this:
+
+```
+git add .
+# This command adds all of the changed files to Git for staging
+git commit -m "A description of my changes"
+# This command commits my changes with a helpful but succinct message
+git push -v 
+# This command will push your changes from your local machine to GitHub
+```
+Using the `-v` or verbose flag should give you feedback in the terminal window after you run the command. 
+
+Once your changes are processed by GitHub, Atlas will automatically begin rebuilding your application based on your latest commit. If you open the [User Portal](https://my.wpengine.com/atlas) in the browser, and navigate to the associated Atlas environment, you can get feedback regarding the build and deployment states of your app. 
+
+![The Atlas build and deployment checklist in a building state](./assets/img/atlas-deploy-checklist-building.png)
+
+Once the app has successfully finished building and deploying, you will see a success message that looks like this:
+
+![The Atlas build and deployment checklist in a building state](./assets/img/atlas-deploy-checklist-success.png)
+
+TODO: Link to Troubleshooting Atlas Build Guide
```

Comment:
Is this intentionally left in?

Reply:
Yup, that is intentional, but thanks for the catch. I've got that stubbed out in a separate branch here: https://github.com/wpengine/headless-docs/commit/506659040175a0d78e538eab32ab6a99965c34fd

They should ship to production at the same time.